code/part-two/client/public/index.html

```diff
@@ -4,6 +4,7 @@
     <title>Cryptomoji</title>
   </head>
   <body>
-
+    <div id="app"></div>
+    <script src="bundle.js"></script>
```

Comment:
ARGH, the reason why I was having trouble w/ React Router's `<BrowserRouter />` was bc I didn't put the `/` before `/bundle.js`. Oops. #noobmistake

Will be fixed via incoming commit.